packages/core/content-type-builder/admin/src/components/ComponentCard/ComponentIcon/ComponentIcon.js

```diff
@@ -2,11 +2,12 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import styled from 'styled-components';
 
-import { Flex } from '@strapi/design-system';
+import { Flex, Icon } from '@strapi/design-system';
+import { Cube } from '@strapi/icons';
+import COMPONENT_ICONS from '../../../utils/componentIcons';
 
 const Wrapper = styled(Flex)`
   border-radius: 50%;
-  color: ${({ theme, isActive }) => (isActive ? theme.colors.primary600 : theme.colors.neutral600)};
   height: ${({ theme }) => theme.spaces[8]};
   width: ${({ theme }) => theme.spaces[8]};
```

Comment:
You can make these props on `Wrapper` instead - `borderRadius` too. This way you don't need the styled overwrite anymore 🪄